mobile/src/screens/ActivityLog.tsx

```diff
@@ -29,11 +33,25 @@ const ActivityLog = ({ navigation, route }: any) => {
   // cancel activity log
   const { isLoading: isCancelingLog, mutate: cancelLog } = useCancelActivityLogMutation();
 
+  const { userProfile } = useUserProfile();
+  const { refetch: reloadCounters } = useActivityLogsCounters(
+    ActivityLogStatus.PENDING,
+    userProfile?.activeOrganization?.volunteerId as string,
+  );
+  const { refetch: reloadActivityLogs } = useActivityLogsInfiniteQuery(
+    OrderDirection.ASC,
```

Comment:
Observation: because the orderDirection and search term are states inside a different screen, I hardcoded them. If you have any other ideas, please let me know 😊